subprojects/docs/src/docs/release/notes.md

```diff
@@ -363,6 +363,12 @@ Several libraries that are used by Gradle have been upgraded:
 - The Maven Wagon libraries used to access Maven repositories have been upgraded from 2.4 to 3.0.0.
 - SLF4J has been upgraded from 1.7.16 to 1.7.25.
 
+### Gradle now bundles JAXB for Java 9 and above
+
+In order to use S3 backed artifact repositories, it was previously required to configure `--add-modules java.xml.bind '-Dorg.gradle.jvmargs=--add-modules java.xml.bind'` in `GRADLE_OPTS` when running on Java 9 and above.
```

Comment:
❌ Not GRADLE_OPTS, but org.gradle.jvmargs. GRADLE_OPTS is for the command line client.

Reply:
Hmm, I copied and pasted that from our official recommendation in https://github.com/gradle/gradle/blob/15b07132343171ba67897c56d6edb36c39be83da/subprojects/resources-s3/src/main/java/org/gradle/internal/resource/transport/aws/s3/S3Client.java#L104-L105

Reply:
Let me break that down:

`GRADLE_OPTS=--add-modules java.xml.bind '-Dorg.gradle.jvmargs=--add-modules java.xml.bind'`

- adds the xml.bind module to the client VM, which is pointless unless you are using --no-daemon, which you shouldn't
- overwrites any VM arguments you specified for your build with `--add-modules java.xml.bind'`, e.g. overwriting your project's memory settings etc.

We should always recommend adding these kinds of things to the project's gradle.properties, not to that environment variable. People generally shouldn't fiddle with the client VM anymore and shouldn't use no-daemon.

Reply:
Makes sense! I'll change it to only mention `org.gradle.jvmargs`.